OWNERS_ALIASES

```diff
@@ -68,3 +68,12 @@ aliases:
   cluster-api-clusterctl-reviewers:
   - fabriziopandini
   - wfernandes
+
+  # -----------------------------------------------------------
+  # OWNER_ALIASES for exp/operator
+  # -----------------------------------------------------------
+
+  cluster-api-operator-reviewers:
+  - alexander-demichev
```

Comment:
is every one in this list a member of k8s-sigs?

Reply:
We will need to sort this out. I am not a member either and might need one more sponsor to get in.

Reply:
WIP here: https://github.com/kubernetes/org/issues/2542

Reply:
I'm not and need one sponsor too.

Reply:
This is all done now :tada: